bip-0045.mediawiki

```diff
@@ -165,27 +165,22 @@ protocol to accept or reject it, belong to another BIP, in my opinion.
 ===Address discovery===
 
 When the master seed is imported from an external source the software should
-start to discover the accounts in the following manner:
+start to discover the addresses in the following manner:
 
-# derive the first account's node (index = 0)
-# derive the external chain node of this account
-# scan addresses of the external chain; respect the gap limit described below
-# if no transactions are found on the external chain stop discovery
-# if there are some transactions, increase the account index and go to step 1
+# for each cosigner:
+# derive the cosigner's node (`m / purpose' / cosigner_index`)
+# for both the external and internal chains on this node (`m / purpose' / cosigner_index / 0` and `m / purpose' / cosigner_index / 1`)
+# scan addresses of the chain; respect the gap limit described below
 
-This algorithm is correct, because software should disallow creation of new
```

Comment:
If you want, you can put in a reminder that, opposite to BIP44, each cosigner branch needs to be checked, even if the earlier ones don't have transactions.